src/psyclone/generator.py

```diff
@@ -282,8 +282,12 @@ def main(args):
         '-I', '--include', default=[], action="append",
         help='path to Fortran INCLUDE or module files')
     parser.add_argument(
-        '-l', '--limit', dest='limit', action='store_true', default=False,
-        help='limit the fortran line length to 132 characters')
+        '-l', '--limit', dest='limit', default='off',
+        choices=['off', 'all', 'output'],
+        help='limit the Fortran line length to 132 characters (default '
+        '\'%(default)s\'). Use \'on\' to apply limit to both input and output '
```

Comment:
All not on?

Reply:
Oops.

Reply:
Again, updated copyright date in this file.